clueless/models.py

```diff
@@ -1,13 +1,163 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-class Room(models.Model):
-	title = models.CharField(max_length=30)
-	background = models.CharField(max_length=50)
-	position = models.IntegerField()
+class Board(models.Model):
+    """
+    Board object for the entire game.  Should be referenced by multiple games and space collections
+    """
+    pass
 
+class SpaceCollection(models.Model):
+    """
+    Any implementations should occupy a set of spaces on the board
+    """
+    board = models.ForeignKey(Board)
+
+class Space(models.Model):
+    """
+    Represents a space on the game board, with an X Y position and all neighbor spaces
+    """
+    posX = models.IntegerField()
+    posY = models.IntegerField()
+    spaceNorth = models.OneToOneField('self', related_name='spaceSouth')
+    spaceEast = models.OneToOneField('self', related_name='spaceWest')
+    spaceCollector = models.ForeignKey(SpaceCollection)
+
```

Comment:
A reference to the space immediately north of the space, and a reference to the space immediately east of the space.  This should make for easy lookups for move validation (lookup space to North and East, reverse lookup space to South and West).